modules/tinymce/src/themes/silver/test/ts/browser/editor/autocomplete/AutocompleterTest.ts

```diff
@@ -563,6 +606,48 @@ UnitTest.asynctest('Editor Autocompleter test', (success, failure) => {
             autocompleteApi.hide();
           }
         });
+
+        ed.ui.registry.addAutocompleter('Card items', {
+          ch: '€',
+          minChars: 1,
+          columns: 1,
+          highlight: [ 'my_title_to_highlight' ],
+          fetch: (pattern, _maxResults) => {
+            const filteredItems = Arr.filter([
+              { text: 'equals sign', value: '=' },
+              { text: 'plus sign', value: '+' }
+            ], (item) => item.text.indexOf(pattern) !== -1);
+            return new Promise((resolve) => {
+              resolve(
+                Arr.map(filteredItems, (item) => ({
+                  value: `euro-${item.value}`,
+                  ariaLabel: item.text,
+                  type: 'cardmenuitem',
+                  onAction: () => {
+                    store.add('euro:AutocompleterContents->onAction');
+                  },
+                  items: [
+                    { type: 'image', src: getGreenImageDataUrl(), classes: [ 'my_autocompleter_avatar_class' ] },
+                    {
+                      type: 'cardcontainer',
+                      direction: 'vertical',
+                      items: [
+                        { type: 'title', text: item.text, classes: [ 'title_class' ], name: 'my_title_to_highlight' },
+                        { type: 'description', text: `Description for ${item.text}`, classes: [ 'description_class' ] }
```

Comment:
After our latest design decision to allow styling with classes, I don't think it makes sense to have more than one text-component. If you need something like a description with smaller text you can achieve that via a class so I'll probably just remove this.

Reply:
I agree!

Reply:
How do you define which field that shows the autocompleter matches...?

Reply:
The way it works with these cards is that the autocompleter now got a `highlight` option (see a few lines above) which will highlight the item(s) with a matching `name`-property. In this test it's the component with the "my_title_to_highlight" name. We had a short discussion about alternatives but this is probably the simplest solution.